### DISCUSSION TITLE
latest/detection/tools/save_detections/

### DISCUSSION THREAD

**Discussion by giscus** [roboflow/supervision#1282] [Docs]
# latest/detection/tools/save_detections/

A set of easy-to-use utilities that will come in handy in any computer vision project.

https://supervision.roboflow.com/latest/detection/tools/save_detections/

**Comment by aberg2323** [comment]


**Reply by LinasKo** [reply]
Hi @aberg2323 :wave: 

`tracker_id` gets added to detections when you call `tracked_detections = byte_tracker.update_with_detections(detections)`

This example may prove useful: https://supervision.roboflow.com/how_to/track_objects/#tracking

**Comment by dearMOMO** [comment]
Hello, according to an mp4 video, generated a csv data, then how to restore these csv data to the video, re-label, frame, color of the box, tracking and other information. Thank you

**Reply by LinasKo** [reply]
Hi @dearMOMO 👋 

Check out this cookbook! It has the steps to save in CSV and convert back to Detections: https://supervision.roboflow.com/develop/notebooks/serialise-detections-to-csv/

**Comment by kennedy-kitoko** [comment]
×

Erreur

X

X

The truth value of an array with more than one element is
ambiguous. Use a.any0 or a.all0

OK
